src/hooks/usePokemon.js

```diff
@@ -1,18 +1,40 @@
 import { useState, useEffect } from 'react';
-import getPokemon from '../pokemon.js';
+import { getPokemon, getTypes } from '../components/pokemon.js';
 
 export default function usePokemon() {
   const [pokemon, setPokemon] = useState([]);
+  const [error, setError] = useState(false);
```

Comment:
Nice use of an error state. I'd suggest initializing as an empty string since there is a chance it may display a message as a string.

Reply:
Changed default to string, error message now shows up at the top of the screen.